Source/SwiftLintFramework/Models/ConfigurationError.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ConfigurationError.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 1/19/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum ConfigurationError: ErrorType {
```

Comment:
Why introduce this if there's only one member, and it's never checked? `ConfigurableRuleType.init(config:)` is only ever used with `try?`, which type-erases the error in a sense.